src/lua/Bootstrapper.h

```diff
@@ -175,31 +167,42 @@ defineSprite(sol::state& target,
     auto spriteType = target.new_usertype<drawing::actors::Sprite>(
       "Sprite",
       sol::factories(
-        []() { return drawing::actors::Sprite::make(); },
         [](sf::Texture& texture) {
             return drawing::actors::Sprite::make(texture);
         },
-        [&textureLoader](const std::string& path) {
+        [&textureLoader](
+          const std::string& path) -> std::shared_ptr<drawing::actors::Sprite> {
             auto* texture = textureLoader.load(path);
             if (texture == nullptr) {
-                return std::shared_ptr<drawing::actors::Sprite>();
+                texture = textureLoader.getFallback();
+            }
+            if (texture == nullptr) {
+                spdlog::error("Fallback texture not found");
+                return nullptr;
             }
             return drawing::actors::Sprite::make(*texture);
         },
         [&textureLoader, eventAttacher, bindAbstractRectLeafProperties](
-          sol::table args) {
-            auto result = drawing::actors::Sprite::make();
+          sol::table args) -> std::shared_ptr<drawing::actors::Sprite> {
+            const auto* texture = static_cast<sf::Texture*>(nullptr);
             if (args["texture"].valid()) {
                 if (args["texture"].get_type() == sol::type::string) {
-                    auto* texture =
+                    texture =
                       textureLoader.load(args.get<std::string>("texture"));
-                    if (texture != nullptr) {
-                        result->setTexture(*texture);
-                    }
                 } else {
-                    result->setTexture(args.get<sf::Texture>("texture"));
+                    texture = args.get<const sf::Texture*>("texture");
                 }
             }
+
+            if (texture == nullptr) {
+                texture = textureLoader.getFallback();
+            }
+            if (texture == nullptr) {
+                spdlog::error("Default texture not found");
+                return nullptr;
+            }
+
+            auto result = drawing::actors::Sprite::make(*texture);
```

Comment:
May want to drop `auto` in places like these as `make` isn't really something super obvious.